lambdas/tests/unit/services/test_send_feedback_service.py

```diff
@@ -0,0 +1,239 @@
+        + "<h2>Overall Experience</h2><p>Very Satisfied</p>"
+        + "</html></body>"
+    )
+
+    actual = send_feedback_service.build_email_body(MOCK_PARSED_FEEDBACK)
+
+    assert actual == expected
+
+
+def test_build_email_body_skip_name_and_email_if_not_given(send_feedback_service):
+    expected = (
+        "<html><body>"
+        + "<h2>Feedback</h2><p>Mock feedback content</p>"
+        + "<h2>Overall Experience</h2><p>Very Satisfied</p>"
+        + "</html></body>"
+    )
+
+    actual = send_feedback_service.build_email_body(MOCK_PARSED_FEEDBACK_ANONYMOUS)
+
+    assert actual == expected
+
+
+def test_send_feedback_by_email_happy_path(send_feedback_service, mock_ses_client):
+    send_feedback_service.ses_client = mock_ses_client
+
+    send_feedback_service.send_feedback_by_email(MOCK_EMAIL_BODY)
+
+    mock_ses_client.send_email.assert_called_with(
+        Source=MOCK_FEEDBACK_SENDER_EMAIL,
+        Destination={"ToAddresses": MOCK_FEEDBACK_RECIPIENT_EMAIL_LIST},
+        Message={
+            "Subject": {"Data": MOCK_FEEDBACK_EMAIL_SUBJECT},
+            "Body": {"Html": {"Data": MOCK_EMAIL_BODY}},
+        },
+    )
+
+
+def test_send_feedback_by_email_raise_error_on_failure(
+    send_feedback_service, mock_ses_client
+):
+    mock_error = ClientError(
+        {
+            "Error": {
+                "Code": "LimitExceededException",
+                "Message": "API call limit exceeded",
+            }
+        },
+        "SendEmail",
+    )
+    mock_ses_client.send_email.side_effect = mock_error
+    send_feedback_service.ses_client = mock_ses_client
+
+    expected_error = SendFeedbackException(500, LambdaError.FeedbackSESFailure)
+    email_body = MOCK_EMAIL_BODY
+
+    with pytest.raises(SendFeedbackException) as error:
+        send_feedback_service.send_feedback_by_email(email_body)
+
+    assert error.value == expected_error
+
+
+def test_get_email_recipients_list_fetch_parameter_from_ssm_param_store(
+    set_env,
+    mock_get_ssm_parameter,
+):
+    mock_get_ssm_parameter.return_value = "gp2gp@localhost, test_email@localhost"
+
+    actual = SendFeedbackService.get_email_recipients_list()
+    expected = ["gp2gp@localhost", "test_email@localhost"]
+
+    assert actual == expected
+
+
+def test_get_email_recipients_list_raise_error_when_fail_to_fetch_from_ssm(
+    set_env,
+    mock_get_ssm_parameter,
+):
+    mock_error = ClientError(
+        {
+            "Error": {
+                "Code": "UnauthorizedException",
+                "Message": "Not authorized to access parameter store",
+            }
+        },
+        "GetParameter",
+    )
+    expected_lambda_error = SendFeedbackException(
+        500, LambdaError.FeedbackFetchParamFailure
+    )
+
+    mock_get_ssm_parameter.side_effect = mock_error
+
+    with pytest.raises(SendFeedbackException) as error:
+        SendFeedbackService.get_email_recipients_list()
+
+    assert error.value == expected_lambda_error
+
+
+@pytest.fixture
```

Comment:
Could we paste these fixtures at the top of the file please just to achieve consistency with other test file structures

Reply:
Of course :D Just did that for both test files for the handler and the service